e2e/StaticLifecycleEvents.test.js

```diff
@@ -83,4 +85,49 @@ describe('static lifecycle events', () => {
       elementByLabel('componentDidAppear | CustomRoundedButton | TopBarButton')
     ).toBeVisible();
   })
+
+  it('disabled back button dispatch event and prevents pop', async () => {
```

Comment:
The title of the test is a bit misleading as the back button isn't disabled. Moreover, these don't check component lifecycle - I think they belong some place else.